app/trustpilot-api/api-bridge.js

```diff
@@ -0,0 +1,41 @@
+module.exports = function (client) {
+
+  async function privateRequest(options) {
+    const requestWrapper = await client.authenticate();
+    return requestWrapper(options);
+  }
+
+  class ApiBridge {
+
+    async getLastUnansweredReview({
+      stars,
+      businessUnitId,
+    }) {
+      const data = await privateRequest({
+        method: 'GET',
+        uri: `/v1/private/business-units/${businessUnitId}/reviews`,
+        qs: {
+          orderBy: 'createdat.desc',
+          responded: false,
+          stars,
+        },
+      });
+      return data.reviews.length > 0 ? data.reviews[0] : null;
+    }
+
+    replyToReview({
+      reviewId,
+      message,
+    }) {
+      return privateRequest({
+        method: 'POST',
+        uri: `/v1/private/reviews/${reviewId}/reply`,
+        form: {
+          message: message,
+        },
+      });
+    }
+  }
+
+  return new ApiBridge();
```

Comment:
Consider refactoring this to use `privateRequest` as a constructor for `ApiBridge`. This currently looks a lot like just a class, but with the constructor method placed outside of the body. Maybe it makes more sense just as a class.